modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/SingleLogoutProfileImpl.java

```diff
@@ -426,6 +402,21 @@ public void terminateSsoSession(
 			SamlWebKeys.SAML_SSO_SESSION_ID, StringPool.BLANK, 0);
 	}
 
+	@Activate
+	protected void activate() {
+		super.setIdentifierGenerationStrategyFactory(
+			_identifierGenerationStrategyFactory);
+
+		super.metadataManager = _metadataManager;
```

Comment:
@slnn For these, if their reference annotations don't have filters (target = xxx), you can put the references in the base class, and enable ds annotation inherit in bnd.bnd. Assigning them in activate method looks unusual.

Reply:
Hi @dantewang 

I have update and forced pushed to this pull/

Thanks
Lily